concent_api/core/message_handlers.py

```diff
@@ -419,22 +411,45 @@ def handle_send_force_get_task_result(client_message: message.concents.ForceGetT
             reason=message.concents.ForceGetTaskResultRejected.REASON.AcceptanceTimeLimitExceeded,
         )
 
-    subtask = store_or_update_subtask(
-        task_id=task_to_compute.compute_task_def['task_id'],
+    subtask = get_one_or_none_subtasks_from_database(
         subtask_id=task_to_compute.compute_task_def['subtask_id'],
-        provider_public_key=provider_public_key,
-        requestor_public_key=requestor_public_key,
-        state=Subtask.SubtaskState.FORCING_RESULT_TRANSFER,
-        next_deadline=(
-            int(task_to_compute.compute_task_def['deadline']) +
-            2 * maximum_download_time +
-            3 * settings.CONCENT_MESSAGING_TIME
-        ),
-        set_next_deadline=True,
-        report_computed_task=client_message.report_computed_task,
-        task_to_compute=task_to_compute,
-        force_get_task_result=client_message,
     )
+    if subtask is None:
+        subtask = store_subtask(
+            task_id=task_to_compute.compute_task_def['task_id'],
+            subtask_id=task_to_compute.compute_task_def['subtask_id'],
+            provider_public_key=provider_public_key,
+            requestor_public_key=requestor_public_key,
+            state=Subtask.SubtaskState.FORCING_RESULT_TRANSFER,
+            next_deadline=(
+                    int(task_to_compute.compute_task_def['deadline']) +
+                    2 * maximum_download_time +
+                    3 * settings.CONCENT_MESSAGING_TIME
+            ),
+            task_to_compute=task_to_compute,
+            report_computed_task=client_message.report_computed_task,
+            force_get_task_result=client_message,
+        )
+
+    else:
+        if subtask.state_enum==Subtask.SubtaskState.FORCING_RESULT_TRANSFER:
```

Comment:
Please fix spaces around `==`.

Reply:
Looks like it's still not done. Have you pushed the latest version of the code?